app/views/tickets/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_with(model: ticket, local: true) do |form| %>
+<%= render 'shared/errors', resource: @ticket%>
+
+  <div class="actions">
+    <%= form.label 'First Name: '%>
+    <%= form.text_field :first_name %>
+    <%= form.label 'Last Name: '%>
+    <%= form.text_field :last_name %>
+    <%= form.label 'Passport Series: '%>
+    <%= form.text_field :passport_series %>
+    <%= form.label 'Passport Number: '%>
+    <%= form.text_field :passport_number %>
+    
+    
+    <%= hidden_field :ticket, :train_id, value: @ticket_params['train_id'] %>
```

Comment:
Не понял зачем тут на какой-то `@ticket_params` полагаться. Почему нельзя эти значение из `ticket` достать?

Reply:
because I need to get those values from url, if I don't get it rom url, then their values are empty on save.
For example, if I don't use @tickets_params 
it will not pass train id

>"ticket"=>{"first_name"=>"Samanta", "last_name"=>"Jones", "passport_series"=>"qq", "passport_number"=>"1234", "train_id"=>"", "start_station_id"=>"1", "end_station_id"=>"3"}, "commit"=>"Create Ticket"}

Reply:
См. коммент в контроллере